scripts/_helpers.py

```diff
@@ -965,105 +965,94 @@ def annuity(n, r):
         return 1 / n
 
 
+# Single source for the currency reference year (aligned with `technology-data` output files / PyPSA-Earth input cost files).
+# Change this value to update the reference year everywhere.
+reference_year = 2020
```

Comment:
Global variables are better fully capitalized; what about making it more tech-data related? feel free to revise as you feel best
```suggestion
TECH_DATA_REFERENCE_YEAR = 2020
```